spec/controllers/v1/inventories_controller_spec.rb

```diff
@@ -0,0 +1,267 @@
+require 'spec_helper'
+require 'faker'
+
+describe V1::InventoriesController do
+  render_views
+
+  describe 'GET #index' do
+    context 'when not authenticated' do
+
+      before(:each) do
+        get :index, format: :json
+      end
+
+      it {
+        expect(response.status).to eq 401
```

Comment:
I personally find `expect(response).to have_http_status :not_found` to be more clear and **cool** :smile: but this is doing the trick too.